ed25519_public_key.go

```diff
@@ -115,10 +119,29 @@ func (pk _Ed25519PublicKey) _BytesRaw() []byte {
 	return pk.keyData
 }
 
-// _Bytes returns the byte slice representation of the _Ed25519PublicKey.
 func (pk _Ed25519PublicKey) _BytesDer() []byte {
-	decoded, _ := hex.DecodeString(ed25519PubKeyPrefix)
-	return append(decoded, pk._BytesRaw()...)
+	type PublicKeyInfo struct {
```

Comment:
Wouldn't it be better to move all asn1 structs (ECPrivateKey, AlgorithmIdentifier, PublicKeyInfo, PrivateKeyInfo) in a separate class?

Reply:
We have a lot of global varaibles already, dont want to bloat it even more.